DataDog/storedog-backend
# Merge frontend repo into backend repo

## Description

This PR brings the https://github.com/DataDog/storedog-frontend repo into this repo.

## Checklist

How to test:
1. Pull down this branch to your local
2. Follow the updated README instructions for local development. 
3. Confirm the app starts up as expected.
